packages/tools/examples/PolySegWasmContourToStackLabelmap/index.ts

```diff
@@ -0,0 +1,245 @@
+import { RenderingEngine, Enums, eventTarget } from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addButtonToToolbar,
+  addDropdownToToolbar,
+  createInfoSection,
+  addManipulationBindings,
+} from '../../../../utils/demo/helpers';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+// This is for debugging purposes
+console.warn(
```

Comment:
This example works the first time now, but does not convert a second time.

Reply:
I will enhance this later to update on any representation edits, when I do the contours